static/js/timerender.js

```diff
@@ -311,12 +311,37 @@ export function get_full_datetime(time) {
     // The sign of getTimezoneOffset() is reversed wrt
     // the conventional meaning of UTC+n / UTC-n
     const tz_offset = -time.getTimezoneOffset() / 60;
+
+    // Since we'll be showing timezone in time string,
+    // (UTC+0) isn't much helpful while we show UTC.
+    const tz_offset_string =
+        tz_offset !== 0 ? " (UTC" + (tz_offset < 0 ? "-" : "+") + tz_offset + ")" : "";
```

Comment:
This is to avoid showing two UTCs here in time strings like: `7:12:53 AM UTC (UTC+0)`